Sync_app/management/commands/create_sales_journal.py

```diff
@@ -0,0 +1,30 @@
+"""Команда запуска синхронизации с МойСклад, КонтурМаркет, GoogleSheets."""
+import datetime
+
+from django.core.management.base import BaseCommand
+
+import Sync_app.konturmarket.konturmarket_class_lib as km_class
+
+
+class Command(BaseCommand):  # noqa: D101
+    help = "Заполнение журнала продаж в КонтурМаркет. "  # noqa
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "date",
+            type=lambda date_: datetime.datetime.strptime(date_, "%Y-%m-%d").date(),
+            nargs='?',
+            default=datetime.datetime.today().strftime('%Y-%m-%d'),
```

Comment:
Я не уверен в какой момент исполняется этот код. Если на старте сервера, то у тебя тут будет ошибка - через день после того как сервер будет запущен today будет не корректным. 

Еще это можно написать красивее
```suggestion
            default=datetime.date.today().isoformat(),
```

Reply:
Я проверил. Если меняю даты в системе и запускаю команду снова, дефолтная дата будет той, что сейчас в системе.